docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,174 @@
+Enable cross-cluster replication in Aiven for Apache Cassandra® |private beta|
+==============================================================================
+
+.. important::
+
+   This private beta feature is available on request only. Contact sales@Aiven.io or support@Aiven.io to opt in. 
+
+This article provides you with instructions on how to set up the cross-cluster replication (CCR) in your Aiven for Apache Cassandra® service using the Aiven console.
+
+About enabling CCR
+------------------
+
+You can enable CCR either when creating a new Aiven for Apache Cassandra service or on an existing service. For that purpose, you can use the following tools:
+
+* `Aiven console <https://console.aiven.io/>`_
+* CLI
+* API
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
+
+  * Access to the `Aiven console <https://console.aiven.io/>`_
+  * `cURL` CLI tool
+  * `Aiven CLI tool <https://github.com/aiven/aiven-client>`_
+
+* Aiven for Cassandra service (only if enabling CCR from an existing service)
+
+Enable CCR in the console
+-------------------------
+
+On a new service
+''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select **+Create service**.
+3. In the **Create service** view
+
+   1. Select Apache Cassandra® and, next, a cloud provider and a cloud region for your service.
+   2. To activate CCR, select the **Enable Cross Cluster Replication** toggle, which unfolds an additional section for setting up another Apache Cassandra service.
+   3. Select a cloud provider and a cloud region for the second service.
+   4. Select one service plan for your two new services.
+   5. Optionally, add an additional disk storage for your services.
+   6. Enter names for your services.
+
+4. Verify your choices in the **Services summary** card on the right and, if your configuration turns out as expected, select **+Create services** if the setting looks.
+
+.. topic:: Result
+   
+   You've created two new services that are connected for CCR purposes. You can preview CCR details for your services in the **Overview** tab under **Cross Cluster replication**.
```

Comment:
From the Aiven side of things that's it, the cluster is CCR-ready; but for the replication to actually work, the user also needs to create a keyspace with correct replication settings; probably a brief mention of this fact + a link to the concept article? otherwise it sounds like "yay, you're done, now everything replicates automagically"